frontend/src/App.tsx

```diff
@@ -1,17 +1,16 @@
 import React from "react";
-import { BrowserRouter, Route, Routes } from "react-router-dom"
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 
-import StyledEngineProvider from '@mui/material/StyledEngineProvider';
-import { ThemeProvider, createTheme } from "@mui/material";
+import { createTheme, ThemeProvider } from "@mui/material";
+import StyledEngineProvider from "@mui/material/StyledEngineProvider";
 
 import { SearchPage } from "./pages/SearchPage";
 
-import "./index.css"
+import "./index.css";
```

Comment:
Styles should be groupped at the bottom of imports

Please connect ESLint & Prettier plugin for automatically groupping of imports by formatter